src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -144,6 +153,7 @@ export class OutlineRegionPicker extends LitElement {
           <input type="radio" id="card-${index}" name="city" value="${index}" ?disabled="${!option.available}" .checked="${this.selectedIndex === index}" @change="${this._locationSelected}">
           <label for="card-${index}" class="city-button">
             <div class="card-header">
+              <div class="tag">${option.lowerCost ? this.localize('region-lowest-cost') : ''}</div>
```

Comment:
It's not possible to remove that dependency, because `GeoLocation` is used throughout the stack.  However, I have removed this field from `CloudLocationOption`, and defined it in a new `RegionPickerOption` instead.